### ISSUE TITLE
check-title CI failed on a valid title 

### ISSUE THREAD

**Issue by delta1** [tari-project/tari#3254]
The valid PR title "feat(base_node): prompt user to create id if not found" failed on this PR: https://github.com/tari-project/tari/pull/3245   

The error message was: 

![image](https://user-images.githubusercontent.com/351403/131087805-632daa0b-646b-4059-bcca-abd92d008412.png)

I suspect that the PR title being passed to "echo" needs to be surrounded with quotes (which worked locally for me)

**Comment by delta1** [comment]
Confirming it's a bash error 

```
# echo feat(base_node): prompt user to create id if not found
bash: syntax error near unexpected token `('
```

**Comment by delta1** [comment]
Fixed in #3273
